tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h

```diff
@@ -50,8 +52,8 @@ DEFINE_REDUCE_UPDATE_OP_GPU(AtomicMax, GpuAtomicMax(dest, value))
 DEFINE_REDUCE_UPDATE_OP_GPU(AtomicMin, GpuAtomicMin(dest, value))
 DEFINE_REDUCE_UPDATE_OP_GPU(NonAtomicSum, *dest += value)
 DEFINE_REDUCE_UPDATE_OP_GPU(NonAtomicProd, *dest *= value)
-DEFINE_REDUCE_UPDATE_OP_GPU(NonAtomicMax, *dest = max(*dest, value))
-DEFINE_REDUCE_UPDATE_OP_GPU(NonAtomicMin, *dest = min(*dest, value))
+DEFINE_REDUCE_UPDATE_OP_GPU(NonAtomicMax, *dest = cuda::std::max(*dest, value))
```

Comment:
This doesn't seem right.  We use `std::max` all the time in CUDA kernels, both in TensorFlow and Eigen.  What exactly is the error this addresses?

... inferring from below, I guess it's trying to pick up the `AlignedVector` type?  We might just be missing a
```
using std::max;
```
declaration.